guidelines/1.1.1.md

```diff
@@ -1,37 +1,60 @@
 
-##### Requirements / What to do?
+## What?
 
-*   Images (like logos and icons) that communicate information have short text descriptions;
-*   Editorial images that support the text around them have short descriptions;
-*   Images (like infographics, charts and diagrams) that communicate complex information also have longer text descriptions within the same page;
-*   Decorative images have empty text descriptions.
+All non-text content like images, charts, icons and infographics, must have an appropriate text equivalent.
 
-##### Common mistakes
+### Requirements
 
-*   The image communicates information but does not have a text description;
-*   The text description does not communicate the same information as the image;
-*   The image has a text description but, like a placeholder or file name, it does not communicate the information in the image;
-*   The image has a text description that is an exact duplication of content elsewhere on the page.
+* Images (like logos and icons) that communicate information have short text descriptions;
+* Editorial images that support the text around them have short descriptions;
+* Images (like infographics, charts and diagrams) that communicate complex information also have longer text descriptions within the same page;
+* Decorative images have empty text descriptions and are not read out by screen readers.
 
-##### Code examples
+### WCAG Success Criteria
 
-###### iOS
+[1.1.1 Non-text content (A)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html)
+
+## Why?
+
+This ensures that information conveyed by non-text content is available to people who cannot see it, like screen reader users.
+
+## How?
+
+### Design
+
+* Define what the text equivalent should be.
+
+#### Supporting docs and references
+
+* [W3C Alt text decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/)
+
+---
+
+### iOS
+
+Giving a **non-decorative** `UIImageView` a label:
 
 ```swift
-// Giving a UIImageView a label
 logoImageView.isAccessibilityElement = true
 logoImageView.accessibilityLabel = "Company Name"
+```
+
+Hiding a **decorative** `UIImageView` from screen readers:
 
-// Hiding a decorative UIImageView
+```swift
 decorativeImageView.isAccessibilityElement = false
 ```
 
-* [UIAccessibility developer reference on apple.com](https://developer.apple.com/documentation/uikit/accessibility/uiaccessibility "developer.apple.com reference")
+#### Supporting docs and references
+
+* [UIAccessibility developer reference](https://developer.apple.com/documentation/uikit/accessibility/uiaccessibility "developer.apple.com reference")
+
+## Further information
 
-##### Useful resources
+### Sources
 
-*   [W3C Requirements for providing text to act as an alternative for images](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)
-*   [W3C Alt text decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/)
+* TODO: Link to GDS
```

Comment:
I struggled to find the right GDS and BBC links, could you give them to me @jfhector and I'll add them in this PR 🙏